tools/prlint/test/lint.test.ts

```diff
@@ -48,6 +48,64 @@ describe('breaking changes format', () => {
   });
 });
 
+describe('ban breaking changes in stable modules', () => {
+  test('breaking change in stable module', async () => {
+    const issue = {
+      title: 'chore(s3): some title',
+      body: 'BREAKING CHANGE: this breaking change',
+      labels: [{ name: 'pr-linter/exempt-test' }, { name: 'pr-linter/exempt-readme' }]
+    };
+    configureMock(issue, undefined);
+    await expect(linter.validatePr(1000)).rejects.toThrow('Breaking changes in stable modules [s3] is disallowed.');
+  });
+
+  test('breaking changes mixed in stable and experimental', async () => {
+    const issue = {
+      title: 'chore(lambda): some title',
+      body: `
+        BREAKING CHANGE: this breaking change
+        continued message
+        * **apigatewayv2**: another breaking change here
+        continued message
+        * **ecs**: further breaking in ecs
+      `,
+      labels: [{ name: 'pr-linter/exempt-test' }, { name: 'pr-linter/exempt-readme' }]
+    };
+    configureMock(issue, undefined);
+    await expect(linter.validatePr(1000)).rejects.toThrow('Breaking changes in stable modules [lambda, ecs] is disallowed.');
+  });
+
+  test('breaking changes only in experimental', async () => {
+    const issue = {
+      title: 'chore(apigatewayv2): some title',
+      body: `
+        BREAKING CHANGE: this breaking change
+        continued message
+        * **lambda-python**: another breaking change here
+      `,
+      labels: [{ name: 'pr-linter/exempt-test' }, { name: 'pr-linter/exempt-readme' }]
+    };
+    configureMock(issue, undefined);
+    await linter.validatePr(1000); // not throw
```

Comment:
same for "valid title" test
```suggestion
    await expect(linter.validatePr(1000)).resolves;
```

Reply:
Nice!